tools/test/config/app_override_libs/targets.json

```diff
@@ -3,7 +3,7 @@
         "core": "Cortex-M0",
         "extra_labels": [],
         "features": [],
-        "default_lib": "std",
+        "c_lib": "std",
```

Comment:
If the build tools default to `std`, you could remove `"c_lib": "std"`. I think we should document it.